src/pages/Settings.tsx

```diff
@@ -82,6 +102,14 @@ function MainSettings() {
           onToggle={toggleHapticFeedback}
         />
       </SettingsSection>
+      <SettingsSection title="Check-out">
+        <SettingToggle
+          title="Check-out"
+          description="Use this app for check-out instead."
```

Comment:
```suggestion
          description="Use this app for check-out instead"
```

This is to keep it consistent with the rest of our setting descriptions.